backend/dal/Services/Concrete/ProjectService.cs

```diff
@@ -595,8 +596,6 @@ public async System.Threading.Tasks.Task RemoveAsync(Project project)
 
             originalProject.Notifications.ForEach(n => this.Context.NotificationQueue.Remove(n));
             originalProject.Notifications.Clear(); // TODO: Need to test this to determine if it'll let us delete a project with existing notifications.
-            originalProject.Notes.ForEach(n => this.Context.ProjectNotes.Remove(n));
```

Comment:
Figured, we don't need this now that I have it set to cascade. Please correct me if I am wrong!